package.json

```diff
@@ -123,7 +123,8 @@
     "ansi-regex": "^5.0.1",
     "@pmmmwh/react-refresh-webpack-plugin": "^0.5.1",
     "postcss": "8",
-    "nanoid": "^3.1.31"
+    "nanoid": "^3.1.31",
+    "minimist": "npm:minimist-lite@2.2.0"
```

Comment:
lol, https://github.com/iTwin/iTwinUI/pull/539
Author released a fix although that repo wasn't touched for 2 years.